app/constants.py

```diff
@@ -3,15 +3,15 @@
 
 from .ntypes import NEWRL_TOKEN_CODE, NUSD_TOKEN_CODE
 
-SOFTWARE_VERSION = "1.3.0"
+SOFTWARE_VERSION = "1.3.1"
```

Comment:
This is a state altering change due to new contract so major version need to change - 1.3.0 -> 1.4.0